src/Common/CommonServices.ts

```diff
@@ -32,7 +32,12 @@ export const enum CommonServiceIds {
     /**
      * Exposes project-related information from the current page
      */
-    ProjectPageService = "ms.vss-tfs-web.tfs-page-data-service"
+    ProjectPageService = "ms.vss-tfs-web.tfs-page-data-service",
+
+    /**
+     * Provides a way to search for identities.
+     */
+    IIVssIdentityService = "ms.vss-features.IIVssIdentityService"
```

Comment:
@thomabr Since there is already an "identity" area, it probably makes sense to define this service name and the interfaces below in a file there rather than in "CommonServices.ts", just to keep it from growing out of control.